src/Subscriptions/Job/BroadcastSubscriptionJob.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+namespace Nuwave\Lighthouse\Subscriptions\Job;
+
+use Illuminate\Bus\Queueable;
+use Illuminate\Contracts\Queue\ShouldQueue;
+use Nuwave\Lighthouse\Schema\Types\GraphQLSubscription;
+use Nuwave\Lighthouse\Subscriptions\Contracts\BroadcastsSubscriptions;
+
+class BroadcastSubscriptionJob implements ShouldQueue
+{
+    use Queueable;
```

Comment:
I decoupled this from the event class in anticipation of its removal in v5. I guess we need to add `SerializesModels` in here?

Reply:
Yes, because of the `$root` argument probably being a model. I've added that trait.